### PR TITLE
[C-990] Nativefy oauth

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Moved lots of things and consolidated code.
OAuth sagas are now entirely on the mobile side because web does things inside the components.
Future TODO to share this code when we try to add tiktok auth as a mechanism to sign up, probably.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Locally with simulator to the extent possible. Will *definitely* need QA because tokens are different for prod apps, etc. etc.

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/src/store/storeContext.ts [hunk 1]
```diff
@@ -45,5 +45,9 @@ export const storeContext: CommonStoreContext = {
   audioPlayer,
   cognito,
   trackDownload,
+  generalAdmission: Config.GENERAL_ADMISSION,
```
Comment (sliptype) [inline packages/mobile/src/store/storeContext.ts:48]: These might be better in the `env` object that lives in storeContext

**Review by sliptype — APPROVED** [review]
Woooo!! Great stuff

File: packages/mobile/src/store/storeContext.ts (hunk 1 above)
Comment (raymondjacobson) [reply packages/mobile/src/store/storeContext.ts:48]: yes! way better. didn't know this was a thing

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-990

**Review by dylanjeffers — APPROVED** [review]
this looks great

File: packages/mobile/src/store/storeContext.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (raymondjacobson) [reply packages/mobile/src/store/storeContext.ts]: has been done! sebastian flagged it

[the remark this answers is not in the document]
Comment (dylanjeffers) [reply packages/mobile/src/store/storeContext.ts]: oh funny, this was a holdover comment from friday that got published when i actually reviewed just now :D

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-990

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-990

**Pull request merged**